docs/browser/events.md

```diff
@@ -0,0 +1,158 @@
+
+<!-- tocstop -->
+
+</details>
+
+This document describes the semantic conventions for events on browser platform. All browser events MUST use a namespace of `browser` in the `event.name` property.
+
+## PageView
+
+The event name MUST be `browser.page_view`.
+
+This event describes the details of the web page visited.
+
+The following table describes the payload fields that MUST be used to describe the details of event.
+
+| Body Field  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `referrer` | string | Referring Page URI (`document.referrer`) whenever available. | `https://en.wikipedia.org/wiki/Main_Page` | Recommended |
+| `type` | int | Browser page type | `0` | Required |
+| `title` | string | Page title DOM property | `Shopping cart page` | Recommended |
+| `url` [1] | string | Full HTTP request URL in the form `scheme://host[:port]/path?query[#fragment]`. Usually the fragment is not transmitted over HTTP, but if it is known, it should be included nevertheless. [2] | `https://en.wikipedia.org/wiki/Main_Page`; `https://en.wikipedia.org/wiki/Main_Page#foo` | Required |
+| `changeState` | string | Type of state change used for the virtual page navigation | `pushState`, `replaceState` | Recommended |
+
+**[1]:**  Alias for [`http.url`](../../../trace/semantic_conventions/http.md)
+**[2]:** The URL fragment may be included for virtual pages
+
+`type` MUST be one of the following:
+
+| Value  | Description |
+|---|---|
+| `0` | physical_page - Initial page load within the browser and will generally also precede a PageNavigationTiming event.|
+| `1` | virtual_page - This is for Single Page Applications (SPA) where the framework provides the ability to perform client side only page "navigation", the exact definition of what a virtual page change is determined by the SPA and the framework it is using.|
+
+## PageNavigationTiming
+
+The event name MUST be`page_navigation_timing`.
+
+This event describes the timing metrics of a page navigation as provided by
+`PerformanceNavigationTiming` Performance API.
+
+The following table describes the payload fields that MUST be used to describe the details of event.
+
+| Body Field  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| name | string | URL of the page || Recommended |
+| fetchStart | long | || Recommended |
+| unloadEventStart | long | || Recommended |
+| unloadEventEnd | long | || Recommended |
+| domInteractive | long | || Recommended |
+| domContentLoadedEventStart | long | || Recommended |
+| domContentLoadedEventEnd | long | || Recommended |
+| domComplete | long | || Recommended |
+| loadEventStart | long | || Recommended |
+| loadEventEnd | long | || Recommended |
+| firstPaint | long | || Recommended |
+| firstContentfulPaint | long | || Recommended |
+
+## ResourceTiming
+
+The event name MUST be `resource_timing`.
+
+This event describes the timing metrics as provided by `PerformanceResourceTiming`
```

Comment:
is there any link that's be useful to share about `PerformanceResourceTiming` - it sounds like something standard and might be worth referring to.

Reply:
This might be useful https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming
and https://www.w3.org/TR/resource-timing/#sec-performanceresourcetiming